tests/astronomy/test_Astronomical.py

```diff
@@ -103,7 +104,21 @@ def test_transit_and_hour_angle():
         )
         / 24
     )
+    invalid = Astronomical.corrected_hour_angle(
+        m0,
+        -36,
+        Coordinates(42.3333, longitude),
+        False,
+        Θ,
+        α2,
+        α1,
+        α3,
+        δ2,
+        δ1,
+        δ3,
+    )
```

Comment:
I'm not sure we should be adding this here. It breaks the current order of the current setup arrange/act/assert of the test for `rise` by adding something not related in between. In fact now that I look at it, I think these tests need refactoring. Initially they were mirroring the original tests in the java codebase but in fact they should be parametrised. https://docs.pytest.org/en/7.3.x/how-to/parametrize.html